recipe/bld_pytorch.bat

```diff
@@ -44,7 +41,19 @@ set "CMAKE_GENERATOR_PLATFORM="
 set "CMAKE_PREFIX_PATH=%LIBRARY_PREFIX%"
 set "libuv_ROOT=%LIBRARY_PREFIX%"
 set "USE_SYSTEM_SLEEF=OFF"
-set "BUILD_CUSTOM_PROTOBUF=OFF"
+set "BUILD_CUSTOM_PROTOBUF=ON"
```

Comment:
```suggestion
@REM Ensure we use OpenMP instead of Intel-OpenMP for compatibility with greater conda-forge ecosystem
set "USE_OPENMP=1"
set "BUILD_CUSTOM_PROTOBUF=ON"
```

Reply:
OpenMP can be disabled with `USE_OPENMP=0` and MKL-DNN can be disabled with `USE_MKLDNN=0`.
`USE_OPENMP=1` (OpenMP support is switched on) is set by default, so this change doesn't do anything.